Assets/UI/Panels/citypanel.lua

```diff
@@ -1468,6 +1604,27 @@ function OnCityMadePurchase(owner:number, cityID:number, plotX:number, plotY:num
     end
 end
 
+
+-- ==== CQUI CUSTOMIZATION BEGIN ====================================================================================== --
+-- CQUI does not implement this function (unmodified defines this function here)
+-- function OnProductionPanelListModeChanged( listMode:number )
+-- ==== CQUI CUSTOMIZATION END ======================================================================================== --
+
+-- ===========================================================================
+function OnCityPanelSetOverViewState( isOpened:boolean )
```

Comment:
these functions are in the unmodified and are what CQUI had been doing inline (changes reflecting as much are below)